.github/workflows/release_debian10.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
 
           # Build community binaries.
           cd build
-          cmake -DCMAKE_BUILD_TYPE=release -DMG_ENTERPRISE=OFF ..
+          cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DMG_ENTERPRISE=OFF ..
```

Comment:
The same mode as we ship to the downloads -> Release at the moment, config

Reply:
Yes because, Jepsen depends on Debian 10, a we build natively on the host os (we have to change that), but this is a good point when we move to a unified runners CC @MarkoBarisic